plugins/orchestrator-backend/package.json

```diff
@@ -60,7 +60,8 @@
     "json-schema": "^0.4.0",
     "openapi-types": "^12.1.3",
     "winston": "^3.11.0",
-    "yn": "^5.0.0"
+    "yn": "^5.0.0",
+    "@urql/core": "^4.1.4"
```

Comment:
FYI we have a library to interact with the data index: `@kie-tools/runtime-tools-gateway-api`.
However, fetching the workflow definition is not available there yet (AFAIK).
cc @paulovmr

Reply:
BTW, this comment is just to let you know that this dependency will probably be removed in the future in favor of `@kie-tools/runtime-tools-gateway-api`. The task to use the new library will be done in https://issues.redhat.com/browse/FLPATH-685